content/docs/command-reference/cache/dir.md

```diff
@@ -32,8 +32,8 @@ cache directory.
 - `--global` - modify the global config file (e.g. `~/.config/dvc/config`)
   instead of the project's `.dvc/config`.
 
-- `--system` - modify the system config file (e.g. `/etc/xdg/dvc/config`) instead
-  of `.dvc/config`.
+- `--system` - modify the system config file (e.g. `/etc/xdg/dvc/config`)
+  instead of `.dvc/config`.
```

Comment:
This one was broken by my formatting of #2311.

Reply:
The other 7 apply the same change to all the refs. as the title of this PR suggests. Sorry for the confusion, should I split this into 2 PRs?